package/src/components/Chip/Chip.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import PropTypes from "prop-types";
+import MuiChip from "@material-ui/core/Chip";
+import makeStyles from "@material-ui/core/styles/makeStyles";
```

Comment:
You can import both `Chip` and `makeStyles` like this: `import { List, withStyles } from '@material-ui/core';`